### ISSUE TITLE
Add textdomain output to 'get code' output.

### ISSUE THREAD

**Issue by menocreativity** [WebDevStudios/custom-post-type-ui#167]
Right now it shows the PHP code to generate the post type.
However, it would be great if you could wrap the labels in __('label','textdomain').
Of course the user (probably a Theme developer) will need to enter the textdomain before generating the code. 

This would be a great addition :) Thanks for the great work! Good luck

**Comment by tw2113** [comment]
@menocreativity not sure what I think of this one at this point, since the values would be pre-set based on labels provided at time of registering a post type or taxonomy. At best, I could see the defaults needing it, but I think in the latest dev version i'm removing any that aren't specifically provided, so default ones from WordPress would be used, and would be part of the core translations.

**Comment by pedro-mendonca** [comment]
@tw2113 is this what you want to do?
https://github.com/pedro-mendonca/custom-post-type-ui/commit/5402b4561ffe4daed1f5b922e33928929552cd25

**Comment by tw2113** [comment]
It's an interesting spot, the "Get Code" section.

In theory, it should end up being already translated one way or another. Let's take Portuguese for example.

If Custom Post Type UI has the Portuguese language files all uptodate, which is does for the most part, then the strings should already be getting translated. This is a combination of WordPress internal defaults for none provided, which I would assume would end up being in Portuguese if the site language is set to that, as well as the code at https://github.com/WebDevStudios/custom-post-type-ui/blob/master/custom-post-type-ui.php#L749:L775.

If the user has provided labels for each spot via our plugin, then that as well will be output internationalized.

If the "Get Code" output is providing English versions, despite either custom labels, default labels, and a Portuguese WP admin language used, then yes we'd need to look into this a bit more. However, I believe the output should be ready to drop in somewhere as-is, without having to re-translate. The only cases I could see not wanting that is if the theme's pot file is desired to take over.

Thoughts from anyone involved with this GH issue?

**Comment by pedro-mendonca** [comment]
Of course that probably the fields should already be in the site main
language, if not by default language chosen in the admin, than by
customizing fields through CPT UI.

The problem is what it does for a multilingual site, and I believe
WordPress is pushing more and more developers to make their plugins/themes
localizable.

So, despite this not being essencial for now, I think it's a good step
forward.

**Comment by tw2113** [comment]
That's where things get murky though. Every hardcoded string that is in the plugin itself is i18n ready. I've had other translators help find missing spots as well, so I'm confident that it's very well covered. This topic though, for the GitHub issue, is relating to generated code.

I'm keeping the issue open as I want others to contribute as well. A question of who's responsible at that point.

**Comment by pedro-mendonca** [comment]
I know that the plugin itself is well localized, I was just reffering to the generated code as asked in the issue.

**Comment by tw2113** [comment]
http://codex.wordpress.org/Function_Reference/wp_get_theme#Get_Other_Data:_Text_Domain_.26_Theme_URI

**Comment by tw2113** [comment]
Despite the fact that I feel it'd be overkill, I have added this.

To clarify my earlier objections.

Say I have plural label of "Movies" in english. If I were to create a German version of this with CPTUI and not fancy tricks, that label would be "Filme". If I went to the Get Code section, "Filme" would already be output in the all the plural label spots, not "Movies". With this change tonight, the output of the plural label spot is going to be as follows:
```
label => __( 'Filme', 'twentysixteen' ),
```
It's already translated because that's the version I put in the settings, and is not:
```
label => __( 'Movies', 'twentysixteen' ),
```
and needing to be re-translated.
Essentially it's dropping in the translated, user-provided values into the __() function. Slight overkill, but I'm not the most worried about the Get Code output as I am over things like things registering properly and the UI not being crap :grinning:
